Logger.Tests/LogFactoryTests.cs

```diff
@@ -1,10 +1,40 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System.IO;
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Logger.Tests;
+
+[TestClass]
+public class LogFactoryTests
+{
+    [TestMethod]
+    public void Configure_NullLogPath_ThrowsArgumentNullException()
+    {
+        LogFactory logFactory = new();
+<<<<<<< HEAD
+        Assert.ThrowsException<ArgumentNullException>(() => logFactory.ConfigureFileLogger(null!));
+    }
 
-namespace Logger.Tests;
-
-[TestClass]
-public class LogFactoryTests
-{
-    
 }
+=======
+        Assert.ThrowsException<ArgumentNullException>(() => logFactory.Configure(null!));
+    }
+
+    [TestMethod]
+    public void Configure_ValidPath_SetsLogFileLocation()
+    {
+        LogFactory logFactory = new();
+
+        string validPath = "LogFile.txt";
+        string res = logFactory.Configure(validPath);
+        Assert.AreEqual(validPath, res);
+    }
+
+    [TestMethod]
+    public void CreateLogger_WithNullLogPath_ReturnsNull()
+    {
+        LogFactory LogFactory = new();
+        LogFactory.Configure(null!);
+    }

```

Comment:
Need to call LogFactory.CreateLogger() to test if an exception is thrown still. Also when using CreateLogger(), you need to pass in the class name of the class that is creating the logger.
```suggestion
    public void CreateLogger_NoConfiguredFileLogger_ReturnsNull()
    {
        LogFactory LogFactory = new();
       BaseLogger? logger = LogFactory.CreateLogger(nameof(LogFactoryTests));
       Assert.isNull(logger);
    }

```

Reply:
awesome thanks for the suggestion!